### PR TITLE
Automatically preview readme files when no file is selected

### LINKED ISSUE #3502: Automatically preview readme files when no file is selected
**Is your feature request related to a problem? Please describe.**
Not a problem, just an idea i had for implementing something from the XP days.

**Describe the solution you'd like**
When in a folder with no file or files selected, the preview pane should preview a file (if it exists) with a specific name, like README.md

**Describe alternatives you've considered**
Only alternative i have come up with is to just click on README.md with the preview pane open

**Additional context**
(This but without me having to select the file)
image

### PR DESCRIPTION
**Pull request by d2dyno1**
**Resolved / Related Issues**
- Closes #3502

**Details of Changes**
- Readme files are now selected when entering a directory. Additionally, replaced `INotifyPropertyChanged` interface in `ItemViewModel` with toolkit's `ObservableObject`

**Validation**
- [x] Built and ran the app

### CODE REVIEW

File: Files/ViewModels/ItemViewModel.cs [hunk 1]
```diff
@@ -36,7 +37,7 @@
 
 namespace Files.ViewModels
 {
-    public class ItemViewModel : INotifyPropertyChanged, IDisposable
+    public class ItemViewModel : ObservableObject, IDisposable
```
Comment (yaichenbaum) [inline Files/ViewModels/ItemViewModel.cs:40]: Is this change related to the current pull request?

**Review by duke7553 — APPROVED** [review]
LGTM

File: Files/ViewModels/ItemViewModel.cs (hunk 1 above)
Comment (d2dyno1) [reply Files/ViewModels/ItemViewModel.cs:40]: > Additionally, replaced INotifyPropertyChanged interface in ItemViewModel with toolkit's ObservableObject

Comment (yaichenbaum) [reply Files/ViewModels/ItemViewModel.cs:40]: I'll let this one pass, but in the future, pull requests will not be approved if they have unrelated changes.

File: Files/ViewModels/ItemViewModel.cs [hunk 2]
```diff
@@ -1117,6 +1117,16 @@ private async void RapidAddItemsToCollectionAsync(string path, string previousDi
                 IsLoadingItems = false;
 
                 AdaptiveLayoutHelpers.PredictLayoutMode(folderSettings, this);
+
+                // Find and select README file
+                foreach (var item in filesAndFolders)
+                {
+                    if (item.ItemName.Contains("readme", StringComparison.InvariantCultureIgnoreCase))
```
Comment (yaichenbaum) [inline Files/ViewModels/ItemViewModel.cs:1124]: Should this be limited to markdown and text files?

Comment (d2dyno1) [reply Files/ViewModels/ItemViewModel.cs:1124]: Then we'd have to check for other formats as well e.g. .rtf
And what a file named "readme" would be used else for in the first place?

Comment (yaichenbaum) [reply Files/ViewModels/ItemViewModel.cs:1124]: I'm not really sure, I guess we can always revisit this.

**Comment by yaichenbaum** [comment]
@d2dyno1 can you adjust the behavior so readme files are only selected when the preview pane is open?
